x-pack/legacy/plugins/monitoring/public/components/elasticsearch/node/node.js

```diff
@@ -42,6 +44,14 @@ export const Node = ({
   return (
     <EuiPage>
       <EuiPageBody>
+        <EuiScreenReaderOnly>
+          <h1>
+            <FormattedMessage
+              id="xpack.monitoring.elasticsearch.node.heading"
+              defaultMessage="Elasticsearch node"
```

Comment:
Should we provide more information about the specific node?

Reply:
Good idea! Done. :)

Reply:
After pushing this change, I am now thinking this might not be viable because the linter doesn't like it:

`ERROR  I18N ERROR  Error in x-pack/legacy/plugins/monitoring/public/components/elasticsearch/node/node.js
      Error: defaultMessage value should be a string or template literal ("xpack.monitoring.elasticsearch.node.heading").`

That indicates to me that we need to use a fixed string and that creating it on-the-fly might break i18n.

Thoughts?

Reply:
@cachedout 
Hmm, strange that the linter would do that. Have you tried it with an arrow function? Or, maybe just do something like:
```Javascript
defaultMessage={`Elasticsearch node ${nodeSummary.name}`}
```

Reply:
Thanks, @igoristic. I implemented your suggestion. Let's see what the tests think. :)